spec/features/admin/add_custom_organisation_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'support/confirmation_use_case'
+
+describe 'adding a custom organisation name' do
+  include_examples 'confirmation use case spy'
+
+  let!(:admin_user) { create(:user, super_admin: true) }
+
+  context 'when visiting the custom organisations page' do
+    before do
+      sign_in_user admin_user
+      visit admin_custom_organisations_path
+    end
+
+    it 'will show the add custom organisations page' do
+      expect(page.body).to have_content('Allow an organisation access to the admin platform')
+    end
+
+    it 'will redirect you to organiastions page once a custom org is added' do
+      fill_in 'custom_organisations[name]', with: 'Custom Org name'
```

Comment:
You can target the label to make specs more readable

"Enter the organisation's full name"